src/components/button/button.vue

```diff
@@ -1,5 +1,5 @@
 <template lang="html">
-  <div :class="{ 'paper-btn': true, 'btn-block': block, 'btn-small': small, 'btn-large': large, 'disabled': disabled }" :disabled="disabled">
+  <div :class="{ 'paper-btn': true, 'btn-block': block, 'btn-small': small, 'btn-large': large, 'disabled': disabled, 'btn-primary': primary, 'btn-secondary': secondary, 'btn-success': success, 'btn-warning': warning, 'btn-danger': danger }" :disabled="disabled">
```

Comment:
Instead of `btn-*`, can't we have a color variable property which takes primary, secondary, ...?

Reply:
Sure